src/hydra_zen/wrapper/_implementations.py

```diff
@@ -1684,6 +1907,22 @@ def get_entry(self, group: GroupName, name: NodeName) -> StoreEntry:
         """
         return _resolve_node(self._internal_repo[(group, name)], copy=True)
 
+    def _set_entry(self, __entry: StoreEntry, overwrite: bool) -> None:
+        _group = __entry["group"]
+        _name = __entry["name"]
+        if not overwrite and (_group, _name) in self._internal_repo:
+            raise ValueError(
+                f"(name={__entry['name']} group={__entry['group']}): "
+                f"Store entry already exists. Use a store initialized "
+                f"with `ZenStore(overwrite_ok=True)` to overwrite config store "
+                f"entries."
+            )
+        self._internal_repo[_group, _name] = __entry
+        self._queue.add((_group, _name))
```

Comment:
should this be `self._queue.add(entry)`?

Reply:
(for posterity)
No, this implementation is correct.